### ISSUE TITLE
azurerm_app_service_certificate_order broken in 2.70.0

### ISSUE THREAD

**Issue by drdamour** [hashicorp/terraform-provider-azurerm#12796]
after upgrading to 2.70 plans/applys are now erroring on existing prior created with certificate orders
does not error with 2.69

```
Error: Error making Read request on AzureRM App Service Certificate Order "op1cer01-tecsys-middleware-maint": web.AppServiceCertificateOrdersClient#Get: Failure responding to request: StatusCode=400 -- Original Error: autorest/azure: Service returned an error. Status=400 Code="NoRegisteredProviderFound" Message="No registered resource provider found for location 'global' and API version '2021-01-15' for type 'certificateOrders'. The supported api-versions are '2020-10-01, 2020-09-01, 2020-06-01, 2019-08-01, 2018-02-01, 2015-08-01'. The supported locations are 'global'."
```

### Community Note

* Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
* Please do not leave "+1" or "me too" comments, they generate extra noise for issue followers and do not help prioritize the request
* If you are interested in working on this issue or have submitted a pull request, please leave a comment

### Terraform (and AzureRM Provider) Version

Terraform v1.0.2
on windows_amd64
+ provider registry.terraform.io/hashicorp/azuread v1.6.0
+ provider registry.terraform.io/hashicorp/azurerm v2.70.0
+ provider registry.terraform.io/hashicorp/null v3.1.0
+ provider registry.terraform.io/microsoft/azuredevops v0.1.6

### Affected Resource(s)

* `azurerm_app_service_certificate_order`

### Terraform Configuration Files

```hcl
resource "azurerm_app_service_certificate_order" "inmp" {
  name                = "op1cer01-inmp"
  resource_group_name = module.certs.resource_group.name
  location            = "global"
  tags                = module.certs.resource_group.tags
  distinguished_name  = "CN=inmp.example.com"
  product_type        = "Standard"
  validity_in_years   = 1
  auto_renew          = true
}
```

### Steps to Reproduce

1. `terraform apply`

### References

* #0000

**Comment by drdamour** [comment]
this may be resolved with 2.77, 2.76 was giving nill pointer errors, but 2.77 seems strong

**Comment by tombuildsstuff** [comment]
Marking this out as closed [since this is apparently fixed in v2.77.0](https://github.com/hashicorp/terraform-provider-azurerm/issues/12796#issuecomment-922108422) - if this is still an issue then please let us know and we can reopen this/take another look.
